src/OnlineRetailPortal.Core/Translators/GetProductsServiceResponseTranslator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OnlineRetailPortal.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Core
+{
+    public static class GetProductsServiceResponseTranslator
+    {
+
+        public static GetProductsServiceResponse ToProductsList(this List<Core.Product> products)
+        {
+            var getProductsResponse = new GetProductsServiceResponse()
+            {
+                Products = GetList(products)
+            };
+            return getProductsResponse;
+
+        }
+
+        private static List<Contracts.Product> GetList(List<Product> products)
```

Comment:
this is giving products list can rename it as GetProductList

Reply:
done